proposal/span.py

```diff
@@ -0,0 +1,27 @@
+from opentracing.span import Span as OTBaseTracer
+
+
+class Span(OTBaseTracer):
```

Comment:
shame on me! :D , wrong module renaming :) it's `OTBaseSpan`, used to be sure that span can be really used with the `basictracer` package (I've a working branch for that).